docs/content/about/introduction.md

```diff
@@ -29,15 +29,15 @@ Automation is a workforce multiplier that helps teams to manage ever growing inf
+  Writing code should be easy. We didn't want to impose a new language, particularly a templating language. We choose the Node.js platform because JavaScript is a widely adopted language, it is not too hard to learn otherwise and its package ecosystem is rich. Internally, we choose CoffeeScript over JavaScript for its expressivity and you are free to choose whichever language you like.
 * Test   
-  We carefully crafted a comprehensive and intuitive API. The code is easy is comprehend and it looks good. It make the writing of tests a pleasant experience, we hope you will agree with us.
+  We carefully crafted a comprehensive and intuitive API. The code is easy is comprehend and it looks good. It makes the writing of tests a pleasant experience, we hope you will agree with us.
 * Versioning   
-  In Nikita, everything is a file. No database to persist states, no agent to deploy and monitor, no server to expose any API. You could implement such dependencies but you are not forced to. Because everything is a file, it is very easy to version you source code with any SCM (Git, Mercurial...) and rely on NPM, the Node.js package manager, to package your code and its release versions.
+  In Nikita, everything is a file. No database to persist states, no agent to deploy and monitor, no server to expose any API. You could implement such dependencies but you are not forced to. Because everything is a file, it is very easy to version your source code with any SCM (Git, Mercurial...) and rely on NPM, the Node.js package manager, to package your code and its release versions.
 * Configure   
-  Passing configuration properties, whether it comes from a flat file, process argument or a database, is super easy, just pass a vanilla JavaScript object to your actions.
+  Passing configuration properties, whether it comes from a flat file, process argument, or a database, is super easy, just pass a vanilla JavaScript object to your actions.
 * Deploy   
-  Because it only rely on files, deploy Nikita is easy. It can fully rely on version control and is quickly integrated to a CI/CD workflow if requested. There is nothing to install on the target machine and Nikita itself will not install anything either for its own purpose. On the host machine, only the Node.js engine to execute the code and NPM to deploy its dependency are expected.
+  Because it only relies on files, deploy Nikita is easy. It can fully rely on version control and is quickly integrated into a CI/CD workflow if requested. There is nothing to install on the target machine and Nikita itself will not install anything either for its purpose. On the host machine, only the Node.js engine to execute the code and NPM to deploy its dependency is expected.
```

Comment:
deploy**ing** Nikita **projects** is easy. **The code** can **entirely** be stored in a version control system like Git and

Reply:
done